source/_integrations/squeezebox.markdown

```
@@ -96,6 +96,51 @@ For example, to play an album from your collection, create an IFTTT applet like
+  command: mixer
+  parameters:
+    - muting
+```
+
+Where a parameter is an increment or decrement, it is necessary to place the value in double quotes.
+
+For example, to increase the playback volume, use the command `mixer` and the parameters `volume` and the amount to increment:
+
+| Row | Parameter | Description |
+| --- | --------  | ----------- |
+|  1  | - volume  | Parameter to change
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)